pyslackers_website/slack/templates/slack/partials/member_map.html

```diff
@@ -0,0 +1,14 @@
+{% load static %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.2.0/leaflet.js"></script>
+<!-- leaflet requires CSS for map applied before init --> 
+<link rel="stylesheet" href="{% static 'css/member_map.css' %}" />
+
+<div class="ui container">
+  <div id='leaflet_member_map' />
```

Comment:
Quoting is inconsistent with the file, please make these double quotes like the rest